includes/wikia/nirvana/WikiaSkin.class.php

```diff
@@ -276,15 +276,17 @@ public function getStylesWithCombinedSASS(Array &$sassFiles) {
 		$sassFiles = $this->assetsManager->getSassFilePath($sassFiles);
 
 		// get a single URL to fetch all the required SASS files
-		$sassFilesUrl = $this->assetsManager->getSassesUrl($sassFiles);
+		$sassParams = SassUtil::getSassSettings();
+		$sassParams['crossorigin'] = 'anonymous';
```

Comment:
Makes sense for me. @drets, @fraszczakszymon, @jacekjursza is this idea going to work with recovery? My assumption is it is since recovery is just a proxy for URLs generated with AM and it does not change the AM groups. What are your thoughts?